### ISSUE TITLE
images not saving to the correct folder

### ISSUE THREAD

**Issue by jing928** [ivyleavedtoadflax/rmd2md#10]
I use `figures` argument to set the directory where the images are saved. However, they are saved to the parent directory instead of the directory I specified. 

For example: 

```r
rmd2md(figures = "./some_folder/target_folder/")
```
Instead of saving my images to the `target_folder`, all my images are saved to the `some_folder`. Also, the names of my images all have a prefix of __"target_folder"__, which is the folder name that I want them to be saved to.

For example, the name of my images are like `target_folderunnamed-chunk-1-1.png`

My laptop is a __Windows 10 64-bit__ machine, and my R version is __3.3.2 64-bit__, and my RStudio version is __1.0.44__.

**Comment by ivyleavedtoadflax** [comment]
Thanks, I have started to look into this, and will solve it in #11.

**Comment by ivyleavedtoadflax** [comment]
This duplicates #8. I've added a build test on appveyor which will test these functions in a windows environment. @mammykins, not entirely sure how https://github.com/ivyleavedtoadflax/rmd2md/commit/b80cd97eb06aeeee8b80bb1f1cd6111c00a75bf9 fixed your issue, as it still seems to be occurring for @jing928 and on windows build tests.

**Comment by ivyleavedtoadflax** [comment]
Have a try with the latest version on the `fix/figure_path` by running `devtools::install_github('ivyleavedtoadflax/rmd2md', ref = 'fix/figure_path')`. This may solve your issue. It's an operating system specific problem, it works fine on mac and linux.

**Comment by jing928** [comment]
Sorry I didn't take a look at the closed issues. 

Thank you so much, I will try it as soon as I get back home (on a short weekend trip) and will let you know if it works.

**Comment by ivyleavedtoadflax** [comment]
Thanks. I find this hard to test because I don't have a windows machine. @mammykins, can you try the above too?

**Comment by mammykins** [comment]
Yea, I will test, I've seen similar problems to what @jing928 experienced

**Comment by mammykins** [comment]
I installed the latest version and it ran OK compared to an earlier version which was throwing an error in regard to the figures dir
`rmd2md::rmd2md()`

I also tried installing with `devtools::install_github('ivyleavedtoadflax/rmd2md', ref = 'fix/figure_path')` and running the default function, again it worked OK. Seems to be sorted on my operating system.

**Comment by ivyleavedtoadflax** [comment]
Just out of interest, can you try setting a different `figures` argument. If it is going to fall over, that is when it will happen...

**Comment by mammykins** [comment]
`rmd2md::rmd2md(figures = "/figures")`

It ignores all the processed files but also produces an error:
`Error in value[[3L]](cond) : 
  Error produced running rmd2md():Error in value[[3L]](cond): unused argument (cond)`

**Comment by ivyleavedtoadflax** [comment]
I've added a test for `figures = /figures`. This passes on linux, but not windows. I'm going to merge #11 for now as there are a lot of incremental updates I want to merge into master. I'll start a new PR to deal with this windows issues.

**Comment by jing928** [comment]
Since you merged the pull request, so I just updated the package and tried the following code on Windows:
```r
rmd2md::rmd2md(figures = "./assets/images/target_folder/")
```

Then I got the same error message as @mammykins ,
```
Error in value[[3L]](cond) : 
  Error produced running rmd2md():Error in value[[3L]](cond): unused argument (cond)
```

**Comment by ivyleavedtoadflax** [comment]
Hmm. I'm going to have to get hold of a windows VM for testing. It's very slow trying to debug the build logs.

@mammykins, is this something you could have a look at fixing?

**Comment by mammykins** [comment]
@ivyleavedtoadflax as Yoda didn't recommend, I will try; currently we have a workaround (from my perspective) so not too urgent

**Comment by ivyleavedtoadflax** [comment]
It sounds like the latest PR will break that workaround. Also my long term goal for this is to move to whole process of compiling and serving on Jekyll to a docker container based on a Linux distro, so unfortunately it's not a high priority for me either.

**Comment by jing928** [comment]
Can I roll back to the previous version, where I could make a workaround?

**Comment by ivyleavedtoadflax** [comment]
Yes, you can just install a previous version with `devtools::install_github('ivyleavedtoadflax/rmd2md', ref=<commit hash>)`

**Comment by mammykins** [comment]
I suggest don't bother fixing then as it just takes me an extra five minutes every couple of weeks when I compile a new post

**Comment by ivyleavedtoadflax** [comment]
I'm now experiencing the error:

```
Error in value[[3L]](cond) : 
  Error produced running rmd2md():Error in value[[3L]](cond): unused argument (cond)
```

which is bizarre because it is passing all the necessary tests. I've reverted to 3434815 in my machine for now, but will have to look at fixing this more seriously at some point.

**Comment by mammykins** [comment]
I'm now on a Mac but encountered the same problem. I tried reversion to fix but this didn't help. While attempting debug I noticed the error was not produced if I defaulted to `dev = "png"`. This lead me to installing [XQuartz](https://www.xquartz.org/) after finding this [help page](http://kshuler.com/blog/getting-svg-graphics-to-work-in-r-on-os-x/). 

I was then able to compile without error (I had to correct the `figures` argument path though as default was spitting them out in teh wrong place)
